contents/docs/billing/common-questions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Common questions about billing
+---
+
+### Can I have a trial?
+
+PostHog is usage-based and you can use it for free for as long as you want (up to 1M events and 15k session replays per month), with no trial needed.
+
+If you have special circumstances where this doesn't work for you, contact our [sales team](/contact-sales). 
+
+### How do I estimate how much PostHog will cost?
+
+See our doc on [estimating usage and costs](/docs/billing/estimating-usage-costs).
+
+### How do I update billing information or see previous months' invoices or usage?
+
+Go to your organization's [billing settings](https://app.posthog.com/organization/billing) and click "Manage card details" to be brought to Stripe where you can update your credit card, billing information, and see your invoice history.
+
+### Are events or replays dropped when I reach billing limits?
```

Comment:
I think we should have another question here of "can I set a billing limit" and then link that to the other doc. Otherwise the next question about data being dropped seems to be missing some context.